.vscode/settings.json

```diff
@@ -2,6 +2,9 @@
 {
     "files.associations": {
         "*.podspec": "ruby",
-        "Podfile": "ruby"
-    }
+        "Podfile": "ruby",
```

Comment:
Interesting. I wasn't sure if I should include the .vscode files, but it's probably easier if we're all using the same environment settings.